app/src/main/java/com/example/poste/models/Post.java

```diff
@@ -27,6 +31,14 @@ public void setUrl(String url) {
         this.url = url;
     }
 
+    public void setTags(String tags) {
+        if (tags.length() < 1) {
+            return;
+        }
+        ArrayList<String> newTagList = new ArrayList<String>(Arrays.asList(tags.split(",")));
```

Comment:
See the comment on User class.
It can be as simple as setTags(List<String> tags)

Reply:
Done.